pinot-tools/src/main/java/org/apache/pinot/tools/perf/QueryRunner.java

```diff
@@ -292,6 +294,7 @@ public static QuerySummary singleThreadedQueryRunner(PerfBenchmarkDriverConf con
     long totalBrokerTime = 0L;
     long totalClientTime = 0L;
     List<Statistics> statisticsList = Collections.singletonList(new Statistics(CLIENT_TIME_STATISTICS));
+    Map<String, QueryStat> queryStatMap = new HashMap<>();
```

Comment:
maybe consider to add a feature flag to enable/disable detailed query stats, in case the number of queries is huge